app/src/main/res/layout/activity_user_profile.xml

```diff
@@ -162,7 +164,31 @@
             android:layout_alignTop="@+id/Button_cancelPhoneEdit"
             android:layout_toStartOf="@+id/Button_cancelPhoneEdit" />
 
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:srcCompat="@drawable/car"
+            android:layout_below="@id/EditText_email"
+            android:layout_alignParentStart="true"
+            android:alpha="255"
+            android:tint="@color/white"
+            android:layout_marginStart="14dp"
+            android:id="@+id/imageView_car"
+            android:scaleX="2"
+            android:scaleY="2" />
+
+        <TextView
+            android:text="Kia Rio"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/imageView_car"
+            android:textColor="@color/white"
+            android:textSize="24sp"
+            android:id="@+id/textView_vehicleDescription" />
```

Comment:
I think you're misunderstanding me. I understand that it will go off the screen in the edittext when you are registering the vehicle and I think that that is expected. The issue is when I go to view the profile and I see the vehicle description I typed out. If it is too long there it gets cut off part way through my sentence. Surrounding the layout that already exists there with a scroll view should fix everything with 2 lines of code. I can show you what I mean later if you like.